sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -1001,10 +1001,13 @@ def test_acos():
     assert acos(-sqrt(2)/2) == pi*Rational(3, 4)
 
     # check round-trip for exact values:
-    for d in [5, 6, 8, 10, 12]:
+    for d in range(5, 13):
         for num in range(d):
             if gcd(num, d) == 1:
                 assert acos(cos(num*pi/d)) == num*pi/d
+                assert acos(-cos(num*pi/d)) == pi - num*pi/d
```

Comment:
Does this fail for `acos(-1.0*cos(num*pi/d))`?

Reply:
I believe it will return the same result as the current code. For example, `acos(-1.0*cos(pi/5))` won't trigger the newly added condition because `acos.eval` receives `-0.25*sqrt(5) - 0.25`.

Reply:
I am intending to ask whether `acos(-1.0*cos(x))` is going to evaluate or not.